geode-apis-compatible-with-redis/src/acceptanceTest/java/org/apache/geode/redis/internal/executor/server/MemoryStatsNativeRedisAcceptanceTest.java

```diff
@@ -31,7 +31,7 @@ public int getPort() {
 
   @Override
   public void configureMemoryAndEvictionPolicy(Jedis jedis) {
-    jedis.configSet("maxmemory", "2000000");
+    jedis.configSet("maxmemory", "18000000");
```

Comment:
Is there any reason we chose 18,000,000?  Locally it passes with 10,000,000. I guess it doesn't really matter though as long as there's no negative impact to having a larger max memory.

Reply:
I tried to match it to the size of the data that's used, but I guess I was a little off.  I can decrease.